lib/basedriver/capabilities.js

```diff
@@ -42,10 +45,10 @@ function validateCaps (caps, constraints = {}, skipPresenceConstraint = false) {
     let message = [];
     for (let [attribute, reasons] of _.toPairs(validationErrors)) {
       for (let reason of reasons) {
-        message.push(` ${attribute} ${reason},`);
+        message.push(`'${attribute}' ${reason}`);
       }
     }
-    throw new Error(message.join(''));
+    throw new Error(message.join(', '));
```

Comment:
perhaps ';' ?

Reply:
Yeah, good call. The messages may have commas in them and that would make things confusing.